test_coverage/schema_validation_testing.py

```diff
@@ -0,0 +1,36 @@
+from app.schema import Mentor, Mentee, Meeting, Feedback
+from data_generators.generators import RandomMentor, RandomMentee, RandomMeeting, RandomMenteeFeedback
+from data_generators.data_options import generate_uuid
+
+
+def test_mentor():
+    assert all(Mentor(**vars(RandomMentor()) for _ in range(1000)))
+
+
+def test_mentee():
+    mentee = RandomMentee()
```

Comment:
This file has old changes from another branch that has since been updated. This needs to be addressed before this branch gets merged.